app/src/main/java/com/example/checknodeavailability/MainActivity.kt

```diff
@@ -8,30 +9,55 @@ import com.jcraft.jsch.JSch
 import android.os.AsyncTask
 import android.util.Log
 import android.widget.Button
+import android.widget.EditText
 import java.io.ByteArrayOutputStream
+import java.io.FileOutputStream
 import java.io.InputStream
 import java.lang.Exception
 
 
 class MainActivity : AppCompatActivity() {
-    val user = System.getenv("username") ?: ""
-    val password = System.getenv("password") ?: ""
-    val host = System.getenv("host") ?: ""
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+        val usernameField = findViewById<EditText>(R.id.username)
+        val IPAddressField = findViewById<EditText>(R.id.ipAddress)
+        val passwordField = findViewById<EditText>(R.id.password)
+        val fileOutputStream:FileOutputStream
+
```

Comment:
Not 100% sure but think you can just write `username` and it'll automatically import the EditTextView. Then you don't have to do `findViewById` at all